wfsim/raw_optical.py

```diff
@@ -74,116 +79,44 @@ def __call__(self, instructions, channels, timings, truth_buffer=None, **kwargs)
             # D) Run all clusters before the current source
             stop_at_this_group = False
             for ibqs in instb_queue:
-                for ptype in [1, 2]:  # S1 S2 PI Gate
+                for ptype in [1, 2, 4, 6]: # S1 S2 PI Gate
```

Comment:
Stupid question, but what are PI and Gate?

Reply:
Afterpulses